client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -646,6 +586,14 @@ export default connect(
 		}
 		const planFeatures =
 			plan && plan.getPlanCompareFeatures ? plan.getPlanCompareFeatures() : false;
+		const jetpackMyPlanPageAvailable = isJetpackMinimumVersion( state, siteId, '6.9-beta' );
+		const jetpackAdminPlansPageUrl = jetpackMyPlanPageAvailable
+			? 'admin.php?page=jetpack#/my-plan'
+			: 'admin.php?page=jetpack#/plans';
+		const jetpackAdminPageUrl = getSiteAdminUrl( state, siteId, jetpackAdminPlansPageUrl );
+		const goBackToSiteLink = isCalypsoStartedConnection
```

Comment:
Two things seem to be wrong with the `isCalypsoStartedConnection` call here:

* `isCalypsoStartedConnection` is a function, and as such needs to be called in order to get a result
* It needs to be called with a site slug, like this: `isCalypsoStartedConnection( siteSlug )`

Let me know if that makes sense.

Reply:
Got it, that explains a few things. My bad for not having looked at the function properly, will fix in a few minutes.